src/blocks/Companies/__stories__/Companies.stories.tsx

```diff
@@ -17,6 +17,16 @@ const DefaultTemplate: StoryFn<CompaniesBlockModel> = (args) => (
     <PageConstructor content={{blocks: [args]}} />
 );
 
+const WithDescriptionTemplate: StoryFn<CompaniesBlockModel> = (args) => (
+    <PageConstructor content={{blocks: [args]}} />
+);
+
 export const Default = DefaultTemplate.bind({});
+export const WithDescription = WithDescriptionTemplate.bind({});
 
 Default.args = data.default.content as CompaniesBlockProps;
+WithDescription.args = {
+    ...data.default.content,
+    description:
+        '<p><strong>Ut enim ad minim veniam</strong> <a href="https://example.com">quis nostrud</a> exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>\n',
```

Comment:
let's add here yfm markdown and yfm supprot as in here: https://github.com/gravity-ui/page-constructor/blob/main/src/blocks/ContentLayout/__stories__/ContentLayout.stories.tsx#L172